test/karma.conf.js

```diff
@@ -0,0 +1,84 @@
+// Karma configuration
+// Generated on Mon Jan 18 2016 19:06:38 GMT+0100 (CET)
+
+module.exports = function(config) {
+  config.set({
```

Comment:
Please format this according to our eslint config. I#ll add another comment in the main PR.